libs/ui-lib/lib/ocm/components/clusterConfiguration/operators/MceCheckbox.tsx

```diff
@@ -23,7 +23,7 @@ const MceLabel = ({
   return (
     <>
       <Tooltip hidden={!disabledReason} content={disabledReason}>
-        <span>Install multicluster engine </span>
+        <span>multicluster engine </span>
```

Comment:
```suggestion
        <span>Multicluster engine </span>
```

In general, we should be using sentence case. See https://www.patternfly.org/ux-writing/capitalization